test/integration/dockerregistryclient_test.go

```diff
@@ -123,7 +123,7 @@ func TestRegistryClientConnectPulpRegistry(t *testing.T) {
 	}, imageNotFoundErrorPatterns...)
 	if err != nil {
 		if strings.Contains(err.Error(), "x509: certificate has expired or is not yet valid") {
-			t.Skip("SKIPPING: due to expired certificate of %s: %v", pulpRegistryName, err)
+			t.Skipf("SKIPPING: due to expired certificate of %s: %v", pulpRegistryName, err)
```

Comment:
Why is this in this pr?

Reply:
This was a drive-by fix; noticed this when building and running the unit tests for this change. Can drop it and follow up with discrete change.